src/schism/schism_esmf_util.F90

```diff
@@ -537,6 +541,8 @@ subroutine SCHISM_StateUpdate1(state, name, farray, kwe, isPtr, rc)
   type(ESMF_Field)               :: field
   real(ESMF_KIND_R8), pointer    :: farrayPtr1(:) => null()
   integer(ESMF_KIND_I4)          :: rc_, localrc, ip
+  integer(ESMF_KIND_I4)          :: ie, indx, ii
```

Comment:
'indx' not well named.  ie and ip are element and node indices, consistent with schism naming,  but indx (and likely ii) should also be semantically named.

Reply:
done